### ISSUE TITLE
Integrate Percy to travis

### ISSUE THREAD

**Issue by erwinmombay** [ampproject/amphtml#8340]

**Comment by rsimha-amp** [comment]
Working on it during this week's sprint.

**Comment by rsimha-amp** [comment]
This issue should ideally be split into 3 issues:

1. Create and configure a Percy account and associate it with ampproject/amphtml
2. Create a new task "gulp visual-diff" and integrate it with the Percy toolchain
3. Integrate "gulp visual-diff" with the Travis CI workflow

Status as of May 1: #1 is complete, #2 is complete* (under final review), and #3 is blocked by additional quota requirements to scale to serve the team's needs.

**Comment by rsimha-amp** [comment]
With the above PRs, this issue can be closed. Scaling up the number of snapshot tests will be tracked in a different issue.
